pairtools/cli/parse2.py

```diff
@@ -43,10 +43,10 @@
     help="""Reported position of alignments in pairs of complex walks (pos columns). 
     Each alignment in .bam/.sam Hi-C-like data has two ends, and you can report one or another depending of the position of alignment on a read or in a pair.  
     
-    "junction" - inner ends of sequential alignments in each pair, aka ligation junctions (complex walks default),
+    "junction" - inner ends of sequential alignments in each pair, aka ligation junctions,
     "read" - 5'-end of alignments relative to R1 or R2 read coordinate system (as in traditional Hi-C),
     "walk" - 5'-end of alignments relative to the whole walk coordinate system,
-    "outer" - outer ends of sequential alignments in each pair. """,
+    "outer" - outer ends of sequential alignments in each pair (parse2 default). """,
```

Comment:
btw, why do we use "outer" as the default? Wouldn't junction make more sense?

Reply:
Ah, "outer" makes the output interpretation similar to traditional Hi-C (as if there pairs were sequences by paired-end seq)!

Reply:
I'll merge it for now (current version keeps the default that we have had already; I've just expanded the docs). Feel free to update anytime if you think it's critical, though!